### PR TITLE
#183898748  Create Sign-Up Page 

### PR DESCRIPTION
**Pull request by RACHID-WEBDEV**
#### What does this PR do?
CompleteCreate Sign-Up Page
#### Description of Task to be completed?

- Integrate Storybook with Tailwind CSS  
- Created  the  `CheckboxGroup Component` 
- Created  the  `HelperLabel Component` 
- Created  the  `Sign-up Page` 
- Created  the  `Forgot-Password  Page` 

#### How should this be manually tested?
- Clone the Repository
- Run `yarn Install` in your terminal to install the required dependencies
-  Run `yarn storybook` to start the local development server on `localhost:3000/sign-up`

### CODE REVIEW

File: components/form/CheckboxGroup.tsx [hunk 1]
```diff
@@ -0,0 +1,164 @@
+import React from "react";
+import { useFormContext } from "react-hook-form";
+import { Label, HelperLabel } from "./Label";
+import { ErrorMessage } from "./ErrorMessage";
+import classNames from "classnames";
+import { FEEDBACK } from "@/utils/constant";
+
+type CheckboxProps = {
```
Comment (andela-hpopoola) [inline components/form/CheckboxGroup.tsx:8]: Please don't forget to add the optional properties here too.

File: components/form/CheckboxGroup.tsx [hunk 2]
```diff
@@ -0,0 +1,164 @@
...
+            "flex items-center",
+            { "pointer-events-none": checked },
+            containerClassName
+          )}
+        >
+          <input
+            checked={checked}
+            disabled={disabled}
+            id={`checkbox-${name}-${value.toString()}`}
+            className={`form-checkbox ${
```
Comment (andela-hpopoola) [inline components/form/CheckboxGroup.tsx]: The logic here looks rough, we could simplify it by properly using the object notation logic for classNames.

**Review by andela-hpopoola — APPROVED** [review]
Thank you for working on this. Do you have any luck with the checkbox accent colors yet?

File: components/form/CheckboxGroup.tsx (hunk 1 above)
Comment (RACHID-WEBDEV) [reply components/form/CheckboxGroup.tsx:8]: Noted Fixed now

File: components/form/CheckboxGroup.tsx (hunk 2 above)
Comment (RACHID-WEBDEV) [reply components/form/CheckboxGroup.tsx]: Noted

**Pull request merged**